conclave-common/src/main/kotlin/com/r3/conclave/common/internal/PluginUtils.kt

```diff
@@ -9,7 +13,23 @@ object PluginUtils {
     const val GRAMINE_MANIFEST = "java.manifest"
     const val GRAMINE_SGX_MANIFEST = "java.manifest.sgx"
     const val GRAMINE_SGX_TOKEN = "java.token"
+    const val GRAMINE_SECCOMP = "gramine-seccomp.json"
+    const val DOCKER_WORKING_DIR = "/project"
```

Comment:
There's no reason why the mounted dir when running the enclave needs to be the same as the mounted directory when the enclave is built. But this is a minor point since we'll be trying to move away from docker at runtime.

Reply:
I agree, but it was convenient I think. I can put a comment actually. Also the `enclave.jar` needs actually to be on the same path both when building and running, but we can leverage the Docker workdir parameter for that.